src/fixed_array.rs

```diff
@@ -0,0 +1,36 @@
+// Copyright 2019-2020 the Deno authors. All rights reserved. MIT license.
+use crate::support::int;
+use crate::Context;
+use crate::Data;
+use crate::FixedArray;
+use crate::HandleScope;
+use crate::Local;
+
+extern "C" {
+  fn v8__FixedArray__Length(this: *const FixedArray) -> int;
+
+  fn v8__FixedArray__Get(
+    this: *const FixedArray,
+    context: *const Context,
+    index: int,
+  ) -> *const Data;
+}
+
+impl FixedArray {
+  pub fn length(&self) -> usize {
+    unsafe { v8__FixedArray__Length(self) as usize }
+  }
+
+  pub fn get<'s>(
+    &self,
+    scope: &mut HandleScope<'s>,
+    index: usize,
+  ) -> Local<'s, Data> {
+    unsafe {
+      scope.cast_local(|sd| {
+        v8__FixedArray__Get(self, sd.get_current_context(), index as int)
```

Comment:
The `index as int` cast is problematic for two reasons:

1. V8 aborts with a runtime error for indices > length
2. Negative indices (which can happen for values > `int::MAX`) result in out-of-bounds reads

My suggestion: check the index against the length first and return `None` if it's not in range.

Reply:
Sure, I'll do that.

Reply:
Done, I left cast to `as int` to make compiler happy